src/index.html

```diff
@@ -8,10 +9,52 @@
 
   <link rel="stylesheet" href="css/main.min.css" />
 </head>
+
 <body>
-  <h1>Hello, World!</h1>
-  <h2>Yarik lox</h2>
+  <header>
+
+    <div class="container">
+      <div class="intro">
+        <div class="circle"></div>
+        <div class="intro_name">
+          <span>Themebau.</span>
+        </div>
```

Comment:
Add support for image logs
Rename `intro_name` to `logo_wrap`
Add class to span